### DISCUSSION TITLE
Manual/Custom performance modes?

### DISCUSSION THREAD

**Discussion by CJKorn** [seerge/g-helper#242] [Ideas]
Would it be possible to implement manual performance modes like in Armoury Crate?

**Comment by paulusgi** [comment]
> Would it be possible to implement manual performance modes like in Armoury Crate?

Hey @CJKorn  It already have it:
![image](https://user-images.githubusercontent.com/102924649/231970386-4c45ec7e-979d-4677-8039-be9839470cc4.png)

Click here to open it:
![Untitled](https://user-images.githubusercontent.com/102924649/231970902-a0fd9a6e-6836-4cb5-a062-606a4e6eae72.png)


Except for the GPU. But for the GPU its a better idea to use MSI Afterburner.

**Reply by CJKorn** [reply]
Oh I was meaning the ability to add more custom fan saved profiles we can set and quickly switch to asides from editing the already existing 3, I guess I should have been more clear.

**Comment by seerge** [comment]
@CJKorn hello. Currently i don't plan to add that as whole UI (and internal app logic) is kind of built on 3 predefined modes with customizations.
